homeassistant/components/apcupsd/coordinator.py

```diff
@@ -19,14 +19,35 @@
     UpdateFailed,
 )
 
-from .const import DOMAIN
+from .const import CONNECTION_TIMEOUT, DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 UPDATE_INTERVAL: Final = timedelta(seconds=60)
 REQUEST_REFRESH_COOLDOWN: Final = 5
 
 
-class APCUPSdCoordinator(DataUpdateCoordinator[OrderedDict[str, str]]):
+class APCUPSdData(OrderedDict[str, str]):
```

Comment:
OrderedDict is not really necessary since a few python releases

Reply:
I think it _was_ more for documentation purposes. But you're right, it's been a while since python 3.7 when this became a language feature instead of a CPython implementation detail.

Updated to use plain `dict` instead. Thanks!